docs/source/manual/jooq.rst

```diff
@@ -112,9 +112,78 @@ Below is an example of configuring the required binding uses a C3P0 pooling data
     }
   }
 
```

Comment:
Add a section why you aren't implementing rollback hooks.

Reply:
https://github.com/cerner/beadledom/pull/140/commits/77d726f9e2db85a48ab8c615eabd5cb20c3e7429